software/control/src/AtlasFallDetector.cpp

```diff
@@ -73,7 +73,7 @@ void AtlasFallDetector::findFootIDS() {
 
 void AtlasFallDetector::handleAtlasStatus(const lcm::ReceiveBuffer* rbuf,
                        const std::string& chan,
-                       const atlas::status_t* msg) {
+                       const drc::behavior_t* msg) {
```

Comment:
I'm confused.  I see some code in this pull request changes atlas::behavior_command_t to drc::behavior_command_t, but this is changing status_t to behavior_t.  What is behavior_t?